### DISCUSSION TITLE
Tags aren't removed by automated rules during import

### DISCUSSION THREAD

**Discussion by kopach** [firefly-iii/firefly-iii#4345] [Firefly III]
**Bug description**
I am running Firefly III version 5.4.5, and my problem is: when importing transaction from `*.csv` file - import tag is added per each transaction (as it should). I have rule with "remove all tags" action to be applied on each transaction create (so also during import). Issue is, that `import` tag isn't removed by rule for some reason.

Basically, I want this rule to remove import tag from every "known" transaction and let me review unknown transactions grouping them by this import tag.

Any ideas what am I doing wrong or how can I get it working?

**Comment by JC5** [comment]
Make sure you disable the setting to add a tag on the import configuration page. It’s not that your rule doesn’t work, it’s just that the tag is added after they transaction is created.
